### PR TITLE
docs: add video description

### PR DESCRIPTION
**Pull request by YangFong**

### CODE REVIEW

File: translations/README-cn.md [hunk 1]
```diff
@@ -216,18 +216,20 @@
...
 ## 不要觉得自己不够聪明
 
 - 大多数成功的软件工程师都非常聪明，但他们都有一种觉得自己不够聪明的不安全感。
-- [天才程序员的神话](https://www.youtube.com/watch?v=0SARbwvhupQ)
-- [不要单打独斗：面对技术中的隐形怪物](https://www.youtube.com/watch?v=1i8ylq4j_EY)
+- 下面的视频可以帮助你克服这种不安全感：
+    - [天才程序员的神话](https://www.youtube.com/watch?v=0SARbwvhupQ)
+    - [不要单打独斗：面对技术中的隐形怪物](https://www.youtube.com/watch?v=1i8ylq4j_EY)
+
+> 译注：观看视频需要访问 YouTube，不知道访问方式的朋友可以看看这个：[free](https://github.com/freefq/free)。
```
Comment (jwasham) [inline translations/README-cn.md:226]: What is this?

Comment (YangFong) [reply translations/README-cn.md:226]: A solution to the inability to access YouTube, If it doesn't feel right, I can delete it.

Comment (jwasham) [reply translations/README-cn.md:226]: Please remove it. It may be a common solution, but let's remove it nonetheless.

**Comment by jwasham** [comment]
Thank you!

**Pull request merged**